defaults.yml

```
@@ -346,6 +346,37 @@ categories:
               height: 40
             text: The time separator flashes every second.
         after: killall SystemUIServer
+      - key: DateFormat
+        domain: com.apple.menuextra.clock
+        title: Set menubar digital clock format
+        description:
+          This setting configures the time and date format for the menubar
+          digital clock. Default value depends on your locale settings. `ss`
+          for seconds, `HH` for 24-hour clock, `EEE` for 3-letter day of the
+          week, `d MMM` for day of the month and 3-letter month.
+          These commands could have a different result depending on the Language & Region parameters.
+        param:
+          type: string
+        examples:
+          - value: "EEE d MMM HH:mm:ss"
```

Fix snippet copy/paste
```suggestion
          - value: '"EEE d MMM HH:mm:ss"'
```